pkg/operation/botanist/seedsystemcomponents/seedadmission/seedadmission.go

```diff
@@ -182,10 +182,15 @@ func (g *gardenerSeedAdmissionController) Deploy(ctx context.Context) error {
 					Spec: corev1.PodSpec{
 						Affinity: &corev1.Affinity{
 							PodAntiAffinity: &corev1.PodAntiAffinity{
-								RequiredDuringSchedulingIgnoredDuringExecution: []corev1.PodAffinityTerm{{
-									TopologyKey:   corev1.LabelHostname,
-									LabelSelector: &metav1.LabelSelector{MatchLabels: getLabels()},
-								}},
+								PreferredDuringSchedulingIgnoredDuringExecution: []corev1.WeightedPodAffinityTerm{
+									{
+										Weight: 1,
```

Comment:
I would probably pick a very high rate because the node with the highest rate gets the Pod, i.e. a low weight might overruled by other criteria during scheduling. WDYT?

Reply:
@mvladev can you double check if this is compliant with your PR https://github.com/gardener/gardener/pull/3243?

Reply:
This PR only affects the scheduling decision for the seed-admission controller itself. #3243 is a completely separate issue.